src/services/projectService.test.ts

```diff
@@ -99,4 +100,17 @@ describe("Project Service", () => {
 
         await expect(projectSerivce.delete(testProject)).rejects.toEqual(expectedError);
     });
+
+    it("isDuplicate returns false when called with a new project", async () => {
```

Comment:
FYI - Test name is a bit off... it can still be a duplicate if it is new if the name and connections are still the same.

Reply:
I changed it to "unique"